src/traces/bar/layout_defaults.js

```diff
@@ -47,4 +47,5 @@ module.exports = function(layoutIn, layoutOut, fullData) {
 
     coerce('bargap', (shouldBeGapless && !gappedAnyway) ? 0 : 0.2);
     coerce('bargroupgap');
+    coerce('barcornerradius');
```

Comment:
We should use `validateCornerradius` to validate the inputs here as well .

Reply:
You are right. It does not seem necessary.
So we could skip it for now and possibly add a line comment?